examples/with-angular/packages/eslint-config/angular.js

```diff
@@ -0,0 +1,68 @@
+const { resolve } = require('node:path');
+
+const project = resolve(process.cwd(), 'tsconfig.app.json');
+
+/**
+ * This is a custom ESLint configuration for use with
+ * Angular applcations
+ */
+
+/** @type {import("eslint").Linter.Config} */
+module.exports = {
+  extends: ['eslint:recommended', 'prettier', 'eslint-config-turbo'],
```

Comment:
```suggestion
  extends: ['eslint:recommended', 'prettier', 'turbo'],
```

This should be only `turbo` instead of `eslint-config-turbo`. This was a bug we had in the rest of the examples, so don't blame you for following what we did.

Make sure to run the `turbo run lint` command to make sure it runs successfully!